### PR TITLE
[#151837054] Add new column free_sms_fragment_limit in the Services table

### PR DESCRIPTION
**Pull request by venusbb**
Prepare the back-end so that we can store different free_sms_fragment_limit values for different organisation types for billing purposes. 

- Add new database column
- Changed schema
- Made sure the existing read and write end points work before the change in the front end

Related Stories
https://www.pivotaltracker.com/story/show/151837054
https://www.pivotaltracker.com/story/show/151176604

### CODE REVIEW

File: environment_test.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ export NOTIFY_ENVIRONMENT='test'
 export ADMIN_CLIENT_SECRET='dev-notify-secret-key'
 export ADMIN_BASE_URL='http://localhost:6012'
 export FROM_NUMBER='from_number'
+export FREE_SMS_TIER_FRAGMENT_COUNT='250000'
```
Comment (servingUpAces) [inline environment_test.sh]: why is this necessary? FREE_SMS_TIER_FRAGMENT_COUNT is in config.py

Comment (klssmith) [reply environment_test.sh]: It wasn't necessary - now removed.

**Comment by venusbb** [comment]
Add new free_sms_fragment_limit column to reflect organisation types changes
